### DISCUSSION TITLE
plant_id ?

### DISCUSSION THREAD

**Discussion by balazs-h** [djansen1987/SAJeSolar#44] [Q&A]
Hello, 
I have problem with installation. 
I did everything according to docs.  What is the solution to avoid plant_id error?

The log sais: 
Invalid config for [sensor.saj_esolar]: [plant_id] is an invalid option for [sensor.saj_esolar]. Check: sensor.saj_esolar->plant_id. (See /config/configuration.yaml, line 23).

I did try these: 
plant_id: 0
no plant id
plant id: 937384729349  (id form the saj website)

**Comment by balazs-h** [comment]
I think ive found. No need to plant_id. 
Other problem: Home assistant energy dashboard does not support most of the data. :( will this improve?

**Comment by djansen1987** [comment]
Hi @balazs-h ,
I think you have installed the beta version. This one is still in development and documentation is not yet ready. You can downgrade via re-download.

Could please give more details on what does not work on the energy dashboard, preferably in an new issues to keep the conversation clear.
